File: frame/src/main/java/com/ssz/frame/utils/LogUtils.java
package com.ssz.frame.utils;

import com.ssz.frame.BuildConfig;

/**
 * @author : zsp
 * time : 2019 08 2019/8/30 16:19
 * logUtils
 */
public class LogUtils {
    private static final boolean DEBUG = BuildConfig.DEBUG;

    public static void i(String tag, String msg) {
        if (DEBUG)
            android.util.Log.i(tag, msg);
    }

    public static void e(String tag, String msg) {
        if (DEBUG)
            android.util.Log.e(tag, msg);
    }

    public static void d(String tag, String msg) {
        if (DEBUG)
            android.util.Log.d(tag, msg);
    }

    public static void v(String tag, String msg) {
        if (DEBUG)
            android.util.Log.v(tag, msg);
    }

    public static void w(String tag, String msg) {
        if (DEBUG)
            android.util.Log.w(tag, msg);
    }
}

File: app/src/main/java/com/ssz/utils/log/LogUtils.kt
package com.ssz.utils.log

import com.ssz.studydemo.BuildConfig


/**
 * @author : zsp
 * time : 2019 09 2019/9/18 15:12
 */
object LogUtils {

    val DEBUG = BuildConfig.DEBUG

    fun i(tag: String, msg: String) {
        if (DEBUG)
            android.util.Log.i(tag, msg)
    }

    fun e(tag: String, msg: String) {
        if (DEBUG)
            android.util.Log.e(tag, msg)
    }

    fun d(tag: String, msg: String) {
        if (DEBUG)
            android.util.Log.d(tag, msg)
    }

    fun v(tag: String, msg: String) {
        if (DEBUG)
            android.util.Log.v(tag, msg)
    }

    fun w(tag: String, msg: String) {
        if (DEBUG)
            android.util.Log.w(tag, msg)
    }
}